[2018-05-21 19:46:04,134] {models.py:167} INFO - Filling up the DagBag from /home/vishwas/PycharmProjects/rtheta_learning/airflow/dags/test_my_operator.py
[2018-05-21 19:46:04,851] {base_task_runner.py:112} INFO - Running: ['bash', '-c', 'airflow run my_test_dag3 unzip_task 2018-05-20T12:00:00 --job_id 23 --raw -sd DAGS_FOLDER/test_my_operator.py']
[2018-05-21 19:46:05,097] {base_task_runner.py:95} INFO - Subtask: [2018-05-21 19:46:05,096] {__init__.py:57} INFO - Using executor SequentialExecutor
[2018-05-21 19:46:05,426] {base_task_runner.py:95} INFO - Subtask: [2018-05-21 19:46:05,425] {models.py:167} INFO - Filling up the DagBag from /home/vishwas/PycharmProjects/rtheta_learning/airflow/dags/test_my_operator.py
[2018-05-21 19:46:06,034] {base_task_runner.py:95} INFO - Subtask: ________________________________________________
[2018-05-21 19:46:06,034] {base_task_runner.py:95} INFO - Subtask: [2018-05-21 19:46:06,034] {models.py:1126} INFO - Dependencies all met for <TaskInstance: my_test_dag3.unzip_task 2018-05-20 12:00:00 [queued]>
[2018-05-21 19:46:06,037] {base_task_runner.py:95} INFO - Subtask: [2018-05-21 19:46:06,037] {models.py:1126} INFO - Dependencies all met for <TaskInstance: my_test_dag3.unzip_task 2018-05-20 12:00:00 [queued]>
[2018-05-21 19:46:06,037] {base_task_runner.py:95} INFO - Subtask: [2018-05-21 19:46:06,037] {models.py:1318} INFO - 
[2018-05-21 19:46:06,037] {base_task_runner.py:95} INFO - Subtask: --------------------------------------------------------------------------------
[2018-05-21 19:46:06,037] {base_task_runner.py:95} INFO - Subtask: Starting attempt 1 of 4
[2018-05-21 19:46:06,038] {base_task_runner.py:95} INFO - Subtask: --------------------------------------------------------------------------------
[2018-05-21 19:46:06,038] {base_task_runner.py:95} INFO - Subtask: 
[2018-05-21 19:46:06,289] {base_task_runner.py:95} INFO - Subtask: [2018-05-21 19:46:06,289] {models.py:1342} INFO - Executing <Task(UnzipOperator): unzip_task> on 2018-05-20 12:00:00
[2018-05-21 19:46:06,303] {base_task_runner.py:95} INFO - Subtask: [2018-05-21 19:46:06,302] {my_operator.py:36} INFO - Hello World!
[2018-05-21 19:46:06,303] {base_task_runner.py:95} INFO - Subtask: [2018-05-21 19:46:06,303] {my_operator.py:37} INFO - operator_param: {'path': '~/output_ORIG.txt.zip'}
[2018-05-21 19:46:06,303] {base_task_runner.py:95} INFO - Subtask: [2018-05-21 19:46:06,303] {models.py:1417} ERROR - [Errno 2] No such file or directory: '~/output_ORIG.txt.zip'
[2018-05-21 19:46:06,304] {base_task_runner.py:95} INFO - Subtask: Traceback (most recent call last):
[2018-05-21 19:46:06,304] {base_task_runner.py:95} INFO - Subtask:   File "/home/vishwas/.pyenv/versions/3.6.5/envs/workflow_intro3/lib/python3.6/site-packages/airflow/models.py", line 1374, in run
[2018-05-21 19:46:06,304] {base_task_runner.py:95} INFO - Subtask:     result = task_copy.execute(context=context)
[2018-05-21 19:46:06,304] {base_task_runner.py:95} INFO - Subtask:   File "/home/vishwas/PycharmProjects/rtheta_learning/airflow/plugins/my_operator.py", line 39, in execute
[2018-05-21 19:46:06,304] {base_task_runner.py:95} INFO - Subtask:     unzip(self.operator_param['path'])
[2018-05-21 19:46:06,304] {base_task_runner.py:95} INFO - Subtask:   File "/home/vishwas/PycharmProjects/rtheta_learning/airflow/plugins/my_operator.py", line 12, in unzip
[2018-05-21 19:46:06,304] {base_task_runner.py:95} INFO - Subtask:     zip_ref = zipfile.ZipFile(path_to_zip, 'r')
[2018-05-21 19:46:06,304] {base_task_runner.py:95} INFO - Subtask:   File "/home/vishwas/.pyenv/versions/3.6.5/lib/python3.6/zipfile.py", line 1090, in __init__
[2018-05-21 19:46:06,304] {base_task_runner.py:95} INFO - Subtask:     self.fp = io.open(file, filemode)
[2018-05-21 19:46:06,304] {base_task_runner.py:95} INFO - Subtask: FileNotFoundError: [Errno 2] No such file or directory: '~/output_ORIG.txt.zip'
[2018-05-21 19:46:06,304] {base_task_runner.py:95} INFO - Subtask: [2018-05-21 19:46:06,304] {models.py:1433} INFO - Marking task as UP_FOR_RETRY
[2018-05-21 19:46:06,399] {base_task_runner.py:95} INFO - Subtask: [2018-05-21 19:46:06,399] {models.py:1462} ERROR - [Errno 2] No such file or directory: '~/output_ORIG.txt.zip'
[2018-05-21 19:46:06,399] {base_task_runner.py:95} INFO - Subtask: /home/vishwas/.pyenv/versions/3.6.5/envs/workflow_intro3/lib/python3.6/site-packages/airflow/ti_deps/deps/base_ti_dep.py:94: DeprecationWarning: generator '_get_dep_statuses' raised StopIteration
[2018-05-21 19:46:06,399] {base_task_runner.py:95} INFO - Subtask:   for dep_status in self._get_dep_statuses(ti, session, dep_context):
[2018-05-21 19:46:06,401] {base_task_runner.py:95} INFO - Subtask: Traceback (most recent call last):
[2018-05-21 19:46:06,402] {base_task_runner.py:95} INFO - Subtask:   File "/home/vishwas/.pyenv/versions/3.6.5/envs/workflow_intro3/bin/airflow", line 28, in <module>
[2018-05-21 19:46:06,402] {base_task_runner.py:95} INFO - Subtask:     args.func(args)
[2018-05-21 19:46:06,402] {base_task_runner.py:95} INFO - Subtask:   File "/home/vishwas/.pyenv/versions/3.6.5/envs/workflow_intro3/lib/python3.6/site-packages/airflow/bin/cli.py", line 422, in run
[2018-05-21 19:46:06,402] {base_task_runner.py:95} INFO - Subtask:     pool=args.pool,
[2018-05-21 19:46:06,402] {base_task_runner.py:95} INFO - Subtask:   File "/home/vishwas/.pyenv/versions/3.6.5/envs/workflow_intro3/lib/python3.6/site-packages/airflow/utils/db.py", line 53, in wrapper
[2018-05-21 19:46:06,402] {base_task_runner.py:95} INFO - Subtask:     result = func(*args, **kwargs)
[2018-05-21 19:46:06,402] {base_task_runner.py:95} INFO - Subtask:   File "/home/vishwas/.pyenv/versions/3.6.5/envs/workflow_intro3/lib/python3.6/site-packages/airflow/models.py", line 1374, in run
[2018-05-21 19:46:06,402] {base_task_runner.py:95} INFO - Subtask:     result = task_copy.execute(context=context)
[2018-05-21 19:46:06,402] {base_task_runner.py:95} INFO - Subtask:   File "/home/vishwas/PycharmProjects/rtheta_learning/airflow/plugins/my_operator.py", line 39, in execute
[2018-05-21 19:46:06,402] {base_task_runner.py:95} INFO - Subtask:     unzip(self.operator_param['path'])
[2018-05-21 19:46:06,403] {base_task_runner.py:95} INFO - Subtask:   File "/home/vishwas/PycharmProjects/rtheta_learning/airflow/plugins/my_operator.py", line 12, in unzip
[2018-05-21 19:46:06,403] {base_task_runner.py:95} INFO - Subtask:     zip_ref = zipfile.ZipFile(path_to_zip, 'r')
[2018-05-21 19:46:06,403] {base_task_runner.py:95} INFO - Subtask:   File "/home/vishwas/.pyenv/versions/3.6.5/lib/python3.6/zipfile.py", line 1090, in __init__
[2018-05-21 19:46:06,403] {base_task_runner.py:95} INFO - Subtask:     self.fp = io.open(file, filemode)
[2018-05-21 19:46:06,403] {base_task_runner.py:95} INFO - Subtask: FileNotFoundError: [Errno 2] No such file or directory: '~/output_ORIG.txt.zip'
[2018-05-21 19:46:09,605] {jobs.py:2083} INFO - Task exited with return code 1
